packages/vitepress-plugin-git-changelog/src/vite/helpers.ts

```diff
@@ -365,17 +365,17 @@ export async function parseCommitAuthors(commit: MergedRawCommit, mapContributor
       if (targetCreatorByName) {
         author.name = targetCreatorByName.name ?? author.name
         author.url = findMapAuthorLink(targetCreatorByName)
-        author.avatarUrl = await newAvatarForAuthor(targetCreatorByName, author_email)
+        author.avatarUrl = await newAvatarForAuthor(targetCreatorByName, author.email!)
```

Comment:
May I ask why changing normal undefinable/nullish value to TypeScript specific non-null assertion operator would help on fixing this？

Reply:
The main change is in the variable name, from `author_email` to `author.email` (`_` -> `.`), the former is the author of the commit (not co-author), the latter is item of the array of `commit authors` + the co-authors from the body.

Before the change, the email of the co-author used the author's instead of its own, thus causing the bug.

The non-null assertion is due to the email is optional in the CommitAuthor type, whereas in the data here, the email must actually exist.

Reply:
> from author_email to author.email (_ -> .)

Ahhh, sorry I didn't notice that. My bad. I understand your change now. Thanks for your kindly explanation.